File: reactor/r2dbc-mysql/src/main/java/ua/mai/zyme/r2dbcmysql/entity/Member.java
package ua.mai.zyme.r2dbcmysql.entity;

import lombok.Builder;
import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Column;


@Builder
@Data
public class Member {

  @Id
  @Column("member_id")
  private Integer memberId;

  private String name;
}

File: kotlin/kotlin-r2dbc-mysql/kotlin-r2dbc-mysql-be/src/main/java/ua/mai/zyme/r2dbcmysql/entity/Member.kt
package ua.mai.zyme.r2dbcmysql.entity

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Column

data class Member @JvmOverloads constructor(
    @Id
    @Column("member_id")
    var memberId: Int? = null,

    var name: String
)